RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -86,3 +86,13 @@ def _updateOutput( era, outputPSets, commands):
                                                                      'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_ZDC_*'
                                                                      ])
                   )
+
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+for ec in [RecoLocalCaloAOD.outputCommands, RecoLocalCaloRECO.outputCommands, RecoLocalCaloFEVT.outputCommands]:
+	pp_on_AA_2018.toModify( ec,
+                  func=lambda outputCommands: outputCommands.extend(['keep *_zdcreco_*_*',
+                                                                     'keep ZDCDataFramesSorted_hcalDigis_*_*',
+                                                                     'keep ZDCDataFramesSorted_castorDigis_*_*',
+                                                                     'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_ZDC_*'
```

Comment:
it seems to me that these are already in the standard RECO and FEVT outputs.

Reply:
You are right. They are defined in the same file.
FEVT: https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py#L20
RECO: 
https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py#L39

Should I revert to the previous commit?

Reply:
Indeed... I did not notice it, sorry.
Yes, either: revert the last commit; or remove from standard RECO and FEVT; or add also to the standard AOD (and remove the customization at all).
I think that reverting the last commit is the simplest (and most obviously back compatible) way of dealing with it.